lib/cartridge/cart.cpp

```diff
@@ -0,0 +1,11 @@
+#include <cartridge/cart.h>
+#include <cartridge/read_file.h>
+#include <cartridge/debug.h>
+
+static cart_context ctx;
+
+bool cart_load(char *cart_path) {
+  read_rom_file(cart_path, &ctx);
```

Comment:
read_rom_file returns false when a error happens and in that case we must exit the program